src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 200px;
```

Comment:
It's better not to hardcode the width. You can use a variable instead. This will make your code more maintainable.